packages/design-tokens/src/tokens/functional/colors/global.json

```diff
@@ -5,46 +5,18 @@
         "fg": {
           "value": "var(--base-color-scale-green-5)",
           "dark": "var(--base-color-scale-green-3)"
-        },
-        "emphasis": {
-          "value": "var(--base-color-scale-green-4)",
-          "dark": "var(--base-color-scale-green-5)"
-        },
-        "muted": {
-          "value": "#4bc36b99",
-          "dark": "#2e9e4299"
-        },
-        "subtle": {
-          "value": "var(--base-color-scale-green-0)",
-          "dark": "#2e9e42d9"
         }
       },
       "error": {
         "fg": {
           "value": "var(--base-color-scale-red-5)",
           "dark": "var(--base-color-scale-red-4)"
-        },
-        "emphasis": {
-          "value": "var(--base-color-scale-red-5)",
-          "dark": "var(--base-color-scale-red-5)"
-        },
-        "muted": {
-          "value": "#ff878599",
-          "dark": "#f8524999"
-        },
-        "subtle": {
-          "value": "var(--base-color-scale-red-0)",
-          "dark": "#f85249d9"
         }
       },
       "accent": {
         "primary": {
           "value": "var(--base-color-scale-pink-5)",
           "dark": "var(--base-color-scale-pink-3)"
-        },
-        "secondary": {
-          "value": "var(--base-color-scale-purple-5)",
-          "dark": "var(--base-color-scale-purple-3)"
```

Comment:
[Looks like we may need additional accent color variables in future](https://turbo-adventure-73kwy3g.pages.github.io/foundations/color) to support our brand pillars, so worth keeping this around?